notebooks/fs.ipynb

```diff
@@ -0,0 +1,1586 @@
+   "id": "7434dee7-fb52-4d9b-b2a1-d414165f3186",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Help on class Copier in module uwtools.fs:\n",
+      "\n",
+      "class Copier(FileStager)\n",
+      " |  Copier(config: Union[dict, str, pathlib.Path, NoneType] = None, target_dir: Union[str, pathlib.Path, NoneType] = None, cycle: Optional[datetime.datetime] = None, leadtime: Optional[datetime.timedelta] = None, keys: Optional[list[str]] = None, dry_run: bool = False) -> None\n",
+      " |\n",
+      " |  Stage files by copying.\n",
+      " |\n",
+      " |  Method resolution order:\n",
+      " |      Copier\n",
+      " |      FileStager\n",
+      " |      Stager\n",
+      " |      abc.ABC\n",
+      " |      builtins.object\n",
+      " |\n",
+      " |  Methods defined here:\n",
+      " |\n",
+      " |  go = __iotaa_tasks__(*args, **kwargs) -> '_AssetT'\n",
+      " |      Copy files.\n",
+      " |\n",
+      " |  ----------------------------------------------------------------------\n",
+      " |  Data and other attributes defined here:\n",
+      " |\n",
+      " |  __abstractmethods__ = frozenset()\n",
+      " |\n",
+      " |  ----------------------------------------------------------------------\n",
+      " |  Methods inherited from Stager:\n",
+      " |\n",
+      " |  __init__(self, config: Union[dict, str, pathlib.Path, NoneType] = None, target_dir: Union[str, pathlib.Path, NoneType] = None, cycle: Optional[datetime.datetime] = None, leadtime: Optional[datetime.timedelta] = None, keys: Optional[list[str]] = None, dry_run: bool = False) -> None\n",
+      " |      Stage files and directories.\n",
+      " |\n",
+      " |      :param config: YAML-file path, or dict (read stdin if missing or None).\n",
+      " |      :param target_dir: Path to target directory.\n",
+      " |      :param cycle: A datetime object to make available for use in the config.\n",
+      " |      :param leadtime: A timedelta object to make available for use in the config.\n",
+      " |      :param keys: YAML keys leading to file dst/src block.\n",
+      " |      :param dry_run: Do not copy files.\n",
+      " |      :raises: UWConfigError if config fails validation.\n",
+      " |\n",
+      " |  ----------------------------------------------------------------------\n",
+      " |  Data descriptors inherited from Stager:\n",
+      " |\n",
+      " |  __dict__\n",
+      " |      dictionary for instance variables\n",
+      " |\n",
+      " |  __weakref__\n",
+      " |      list of weak references to the object\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "help(fs.Copier)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "061ac341-96cb-4af6-94ce-4f1e4d342b63",
+   "metadata": {},
+   "source": [
+    "A `Copier` class is instantiated using the same parameters as `copy()`, but copying is not performed until `Copier.go()` is called."
```

Comment:
```suggestion
    "A `Copier` object is instantiated using the same parameters as `copy()`, but copying is not performed until `Copier.go()` is called."
```
In OO terms, an object (or instance) is instantiated from a class. "Instance" is probably the better term, but "instance is instantiated" sounds awkward.